src/client/i18n/de.po

```diff
@@ -52,32 +54,32 @@ msgstr "ÖPNV und Fahrrad"
 #. Options widgets)
 #: src/client/js/otp/config.js:293
 msgid "Walk Only"
-msgstr "zu Fuß"
+msgstr "nur zu Fuß"
 
 #. Travel by: mode of transport (Used in selection in Travel
 #. Options widgets)
 #: src/client/js/otp/config.js:296
 msgid "Drive Only"
-msgstr "Auto"
+msgstr "nur Auto"
 
 #. Travel by: mode of transport (Used in selection in Travel
 #. Options widgets)
 #: src/client/js/otp/config.js:299
 msgid "Park and Ride"
-msgstr "Parken und Reisen"
+msgstr "Parken and Ride"
```

Comment:
I can see that the English terms are commonly used in German speaking areas (Kiss and Ride etc.) but is this German-English hybrid "Parken and Ride" a mistake?

Reply:
Yes it should be "Park and Ride"